### PR TITLE
initial browser tests for geo type statements and properties  T88543

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
https://phabricator.wikimedia.org/T88543

### CODE REVIEW

**Comment by zeljkofilipin** [comment]
:+1:

File: tests/browser/features/statements_geo.feature [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    | 1 1 | 1°N, 1°E | ±1° |
+    | 1 S 1 W | 1°S, 1°W | ±1° |
+    | 52°29'53"N, 13°22'51"E | 52°29'53"N, 13°22'51"E | to an arcsecond |
+    | 52°29'N, 13°22'E | 52°29'N, 13°22'E | to an arcminute |
+    | 42.1538, 8.5731 | 42°9'13.7"N, 8°34'23.2"E | ±0.0001° |
+    | 42° 09.231' N 008° 34.386' E | 42°9'13.86"N, 8°34'23.16"E | to 1/100 of an arcsecond |
+
+  @repo_login @modify_entity
+  Scenario: Adding a statement of type geo
+    Given I am logged in to the repo
```
Comment (thiemowmde) [inline tests/browser/features/statements_geo.feature]: Why do the last 2 tests need login but the others do not?

**Comment by thiemowmde** [comment]
Another +1, code looks good, just a question.

File: tests/browser/features/statements_geo.feature (hunk 1 above)
Comment (WMDE-Fisch) [inline tests/browser/features/statements_geo.feature]: because changing a statement of that type is considered "harmfull" on beta

Comment (thiemowmde) [inline tests/browser/features/statements_geo.feature]: The spam protection going nuts? Ah dang...

**Pull request merged**